includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -263,6 +263,7 @@ public function create_and_confirm_intention(
 	 * @param string      $capture_method  - optional capture method (either `automatic` or `manual`).
 	 * @param array       $metadata        - A list of intent metadata.
 	 * @param string|null $customer_id     - Customer id for intent.
+	 * @param array       $additional_parameters  - An array of any additional request parameters, particularly for additional payment methods.
```

Comment:
nitpick: I see that doc comments are indented in three columns above, this breaks the rule 😅  But again, it's not important, the linter is not complaining

Reply:
Thanks for noticing it, I would like it to be fixed by PHPCBF! Addressed in d220c6d